examples/r2i/edgetpu/Makefile.am

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2020 RidgeRun, LLC (http://www.ridgerun.com)
+# All Rights Reserved.
+#
+# The contents of this software are proprietary and confidential to RidgeRun,
+# LLC.  No part of this program may be photocopied, reproduced or translated
+# into another programming language without prior written consent of
+# RidgeRun, LLC.  The user is free to modify the source code after obtaining
+# a software license from RidgeRun.  All source code changes must be provided
+# back to RidgeRun without any encumbrance.
+
+AM_DEFAULT_SOURCE_EXT = .cc
+
+if ENABLE_EXAMPLES
+
+noinst_PROGRAMS =                  \
+        inception
+
+AM_CXXFLAGS =                      \
+        $(RR_CXXFLAGS)             \
+        $(CODE_COVERAGE_CXXFLAGS)  \
+        -I../common/
+
+AM_CFLAGS =                        \
+        $(RR_CFLAGS)               \
+        $(CODE_COVERAGE_CFLAGS)
+
+AM_CPPFLAGS =                      \
+        $(RR_CPPFLAGS)             \
+        $(CODE_COVERAGE_CPPFLAGS)
+
+LDADD =                            \
+        $(RR_LIBS)                 \
+        $(CODE_COVERAGE_LIBS)      \
+        $(TFLITE_LIBS)             \
+        $(EDGETPU_LIBS)            \
```

Comment:
None of these should be necessary. they should be absorbed by our lib tool library